api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
Why do we have this? No update of `controller-gen` dependency, wondering why CI is not complaining, too.

Reply:
seems like something with my local `controller-gen`. I've removed this changes